docker-for-azure/index.md

```diff
@@ -11,20 +11,26 @@ redirect_from:
 ## Quickstart
 
 If your account has the [proper permissions](#prerequisites), you can generate the [Service Principal](#service-principal) and
-then use the below button from the stable channel to bootstrap Docker for Azure using Azure Resource Manager.
+then use the below button from the stable or beta channel to bootstrap Docker for Azure using Azure Resource Manager. 
```

Comment:
maybe "Choose the stable or beta channel" rather than "Use the button below", it should be pretty obvious, right?